docs/quickstart.md

```diff
@@ -1,24 +1,55 @@
 To get up and going with the DAQ basics, all you need to do is...
 
+## Install
 (starting with a vanilla Debian Linux install)
 <pre>
 # git clone http://github.com/faucetsdn/daq.git
 &hellip;
-# cd daq
-# bin/setup_daq
+cd daq
```

Comment:
The # at the front was the distinguisher from "command prompt" -- if not #, I think it should be something... otherwise you can't tell the difference from command output.

Reply:
i think I'll use a $ instead. # is more like a comment to me.